internal/cmd/test.go

```diff
@@ -135,3 +137,43 @@ func runCMD(mainFunc func(), args []string) (err error) {
 
 	return err
 }
+
+type CommonCMDTestCases struct {
+	ExpectedExitCode int
+	ExpectedStdout   string
+	ExpectedStderr   string
+}
+
+func RunCommonCMDTests(test *testing.T, mainFunc func(), args []string, commonCases CommonCMDTestCases, prefix string) (string, string, int) {
```

Comment:
This function needs a final return, an "ok" boolean that signals if an error occurred.
This will allow tests to decide if they want to move on to the next test case.